src/pods/user-settings/user-settings.vm.ts

```diff
@@ -0,0 +1,4 @@
+export interface UserSettings {
+  numberQuestions: number;
+  secondChance: boolean;
+}
```

Comment:
If we define a VM we will need a mapper

Reply:
In this case we could just use the global Entity, as you wish

Reply:
Yep I was already using the global one, lost track of this file. Deleting.